### ISSUE TITLE
setup bmbridge as a windows service

### ISSUE THREAD

**Issue by jaytaph** [bitmaelum/bitmaelum-suite#184]
Allow bmbridge to run as a service on windows (but likely on other oses as well

https://billg.sqlteam.com/2018/07/17/running-go-as-a-windows-service/

**Comment by acalatrava** [comment]
I implemented service for both bm-server and bm-bridge. However bm-bridge is not working properly because of vault password (not sure why yet)

There is a new command on bm-config to manage services

```
#./bm-config service
Service management

Usage:
  bm-config service [command]

Available Commands:
  install     Installs the service bm-server into the system
  remove      Remove the service from the system
  start       Starts the service
  stop        Stops the service

Flags:
  -h, --help   help for service
  ```

```
#./bm-config service install
Installs the service bm-server into the system

Usage:
  bm-config service install [flags]

Flags:
      --bm-bridge         Manage bm-bridge service
      --bm-server         Manage bm-server service
  -h, --help              help for install
      --imaphost string   Set the host for the IMAP server (bm-bridge)
      --smtphost string   Set the host for the SMTP server (bm-bridge)
```

https://github.com/bitmaelum/bitmaelum-suite/tree/bm-server-service

**Comment by acalatrava** [comment]
Fixed on PR #191
